### DISCUSSION TITLE
ERROR: Sign in to confirm you’re not a bot. This helps protect our community. Learn more

### DISCUSSION THREAD

**Discussion by mikieee25** [deepjyoti30/ytmdl#301] [Q&A]
What to do when this happens?

**Comment by deepjyoti30** [comment]
This is not an issue of ytmdl but instead coming from yt-dlp and the likely reason is that your IP is blocked by YouTube.

There is an open discussion on this on yt-dlp: https://github.com/yt-dlp/yt-dlp/issues/10128

The only solution that I have seen working to get around this issue is using the oauth approach which is mentioned here: https://github.com/yt-dlp/yt-dlp/issues/10128#issuecomment-2157140324
